eclair-core/src/test/scala/fr/acinq/eclair/integration/ChannelIntegrationSpec.scala

```diff
@@ -623,9 +624,17 @@ class StandardChannelIntegrationSpec extends ChannelIntegrationSpec {
     bitcoinClient.publishTransaction(revokedCommitTx).pipeTo(sender.ref)
     sender.expectMsg(revokedCommitTx.txid)
     bitcoinClient.publishTransaction(htlcSuccess.head).pipeTo(sender.ref)
-    sender.expectMsg(htlcSuccess.head.txid)
+    sender.expectMsgType[Any] match {
+      case txid: TxId => assert(txid == htlcSuccess.head.txid)
+      // 3rd stage txs (txs spending htlc txs) are not tested if C publishes htlc-success before F
```

Comment:
We have a specific name for the case where we claim an HTLC output directly from the revoked commitment (instead of claiming the output of a revoked HTLC transaction), it's clearer if we use those.

```suggestion
      // 3rd stage txs (txs spending htlc txs) are not tested if C publishes the htlc-penalty transaction before F publishes its htlc-success
```

Reply:
Fixed in [fbf6939](https://github.com/ACINQ/eclair/pull/2871/commits/fbf6939fd83099f608f6a88c51a5428e01602cd6).